genrl/agents/deep/base/offpolicy.py

```diff
@@ -37,7 +39,7 @@ class OffPolicyAgent(BaseAgent):
     """
 
     def __init__(
-        self, *args, replay_size: int = 5000, buffer_type: str = "push", **kwargs
+        self, *args, replay_size: int = 5000, buffer_type: str = "reverb", **kwargs
```

Comment:
Maybe not leave this on by default? Most usage at least for now will probably be on simple envs. Besides we don't have distributed stuff yet

Reply:
Hmm yeah, this was just for testing. Will change it to `push`